.env.development

```diff
@@ -32,4 +32,5 @@ APPLE_PAY_MERCHANT_CAPABILITIES='supports3DS,supportsCredit,supportsDebit',
 STRIPE_API_VERSION=2022-08-01;server_side_confirmation_beta=v1
 STRIPE_BETA_FLAG=server_side_confirmation_beta_1
 STRIPE_PUBLISHABLE_KEY=null
+STRIPE_DEFERRED_INTENT_BETA_FLAG=elements_enable_deferred_intent_beta_1
```

Comment:
Just a reminder to do the `edx-internal` changes for this beta flag as well, for stage and prod yml files

Reply:
Ok just saw your TODO comment in another file, thank you!